data/json/monsters/reptile_amphibian.json

```diff
@@ -341,8 +341,9 @@
     "id": "mon_toad",
     "type": "MONSTER",
     "name": { "str": "toad" },
-    "description": "Toads are a diverse and varied family of amphibians sister to frogs.  Very distinguishable for their warty and dry skin.",
+    "description": "Toads are a largely colloquial grouping of frogs.  Distinguished by their warty and dry skin.",
     "copy-from": "mon_frog",
+    "upgrades": { "half_life": 34, "into": "mon_toad_small" },
```

Comment:
Don't use straight evolution for mutants - that's a zombie thing. Either set up the groups to spawn them occasionally later and/or do a `base->egg->base or weird` progression in their reproduction.

Reply:
I'm uncertain how to implement this. The "upgrades" line was added to mimic the same in the existing "mon_frog" entry, and also overwrite its "into" value from the "copy-from."
It looks as though the existing .json had (normal) toads and frogs both upgrading to mutant bullfrogs. My intent was to redirect the toads to their own existing mutant versions and add the beaked toad to the end of that line, akin to the frog mutants:
tadpole -> frog -> mutant bullfrog -> giant bullfrog -> colossal croaker
brown tadpole -> toad -> huge toad -> giant toad -> great beaked toad

I am looking into adding the "mon_toad_beak" to group spawns as well

Reply:
If you use evolution they won't exist as tadpoles ever - evo clock starts at game start, so after X time fully-formed mutanta would pop out of eggs. If you want them to *grow* you want to use `age_grow`, and handle world progression by their relevant spawn monster groups.

For a reference of what I mean check out the cellar spiders as an example.